test/unit/api/subscriptions.spec.ts

```diff
@@ -255,18 +255,39 @@ describe('Subscriptions Tests', () => {
       expect(thread).to.not.be.null;
     });
 
-    it('/viewNotifications', async () => {
-      // Get Notifications for Default User
-      expect(subscription).to.not.be.null;
-      expect(thread).to.not.be.null;
-      const res = await chai.request(app)
-        .post('/api/viewNotifications')
-        .set('Accept', 'application/json')
-        .send({ jwt: jwtToken });
-      expect(res.body).to.not.be.null;
-      expect(res.body.status).to.be.equal('Success');
-      expect(res.body.result.length).to.be.greaterThan(0);
-      notifications = res.body.result;
+    describe('/viewNotifications route tests', () => {
```

Comment:
here too

Reply:
"/viewNotifications: return notifications to user"